joss/paper.md

```diff
@@ -44,9 +44,11 @@ Some optimal control libraries tackle this numerical challenge with a monolithic
 While effective, the monolithic nature is both less efficient[@mader2008adjoint; @kenway2019effective] and less flexible.
 
 Instead, Dymos uses a modular approach that allows users to select any combination of finite-difference, complex-step [@Martins2003CS], algorithmic differentiation, and hand differentiation that suits their modeling needs.
-This flexibility is achieved by leveraging the data passing and efficient differentiation fearures of NASA's OpenMDAO framework[@Gray2019a]. 
+This flexibility is achieved by leveraging the data passing and efficient differentiation features of NASA's OpenMDAO framework[@Gray2019a].
+OpenMDAO is a Python-based implementation of a framework for evaluating derivatives across complex, coupled multidisciplinary models developed by Hwang and Martins [@hwang2018b].
```

Comment:
`models developed by Hwang and Martins`

This sentence technically states that OpenMDAO only works on models that were developed by H&M.  The implied meaning is that OpenMDAO is a tool developed by H&M.  I think you really want to say that the framework's  derivatives capability was based on H&M's pioneering research with the unified derivatives methods.